### PR TITLE
add redirect for cli install using windows powershell script

### PR DESCRIPTION
**Pull request by choir27**
Adds redirect for cli install using windows powershell script

## What does this PR do?

Fixes [#133](https://github.com/appwrite/sdk-for-cli/issues/133)
 
## Test Plan

X

## Related PRs and Issues

- https://github.com/appwrite/sdk-for-cli/issues/133

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/redirects.json [hunk 1]
```diff
@@ -675,6 +675,10 @@
         "link": "/policy/privacy",
         "redirect": "/privacy"
     },
+    {
+        "link": "/cli/install.sh",
+        "redirect": "https://appwrite.io/cli/install.ps1"
```
Comment (stnguyen90) [inline src/redirects.json:680]: Did this work for you on windows?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: src/redirects.json (hunk 1 above)
Comment (choir27) [reply src/redirects.json:680]: It did not, but after changing it to the following: 
```
{
   "link": "/cli/install.ps1",
   "redirect": "https://raw.githubusercontent.com/appwrite/sdk-for-cli/master/install.ps1"
}
```
I recieve the following error message on testing the windows powershell script cli install command:
```
iwr -useb http://localhost:5173/cli/install.ps1 | iex
iwr : The remote server returned an error: (308) Permanent Redirect.
At line:1 char:1
+ iwr -useb http://localhost:5173/cli/install.ps1 | iex
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
```
So I tried to make sure `curl -sL https://appwrite.io/cli/install.sh | bash` was working properly. But I also recieve the following error message:
```
$ curl -sL https://appwrite.io/cli/install.sh | bash
     _                            _ _           ___   __   _____
    /_\  _ __  _ ____      ___ __(_) |_ ___    / __\ / /   \_   \
   //_\\| '_ \| '_ \ \ /\ / / '__| | __/ _ \  / /   / /     / /\/
  /  _  \ |_) | |_) \ V  V /| |  | | ||  __/ / /___/ /___/\/ /_
  \_/ \_/ .__/| .__/ \_/\_/ |_|  |_|\__\___| \____/\____/\____/
        |_|   |_|


🔥 Welcome to the Appwrite CLI install shield 🛡
[1/4] Getting System Info ...

OS : mingw64_nt-10.0-22631
ARCH : x64
REQUIRES ROOT : false

[2/4] Downloading executable for mingw64_nt-10.0-22631 (x64) ...
🚦 Downloading Appwrite CLI 5.0.5 ...
❌ Couldn't find executable for mingw64_nt-10.0-22631 (x64). Please contact the Appwrite team
```

Comment (stnguyen90) [reply src/redirects.json:680]: The link shouldn't be install.sh.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Comment by choir27** [comment]
Changed status: 308 to status: 301 in order to make sure `iwr -useb https://appwrite.io/cli/install.ps1 | iex` works properly for this PR

**Comment by stnguyen90** [comment]
After talking to @TorstenDittmann, we want to have a proper redirect using a +page.ts endpoint because this redirects.json was only intended to be used for backwards compatibility for old links (Mainly because they are not checked during build).

**Pull request closed without merging**